### PR TITLE
Fix error message on loading file with chart

### PR DESCRIPTION
**Pull request by fur-gan**

### CODE REVIEW

**Comment by DmitryAstafyev** [comment]
@fur-gan Hi!
I **cannot** confirm an issue. Tested on mac/linux/windows - all works.

File: application/client.core/src/tsconfig.app.json [hunk 1]
```diff
@@ -2,7 +2,8 @@
   "extends": "../tsconfig.json",
   "compilerOptions": {
     "outDir": "../out-tsc/app",
-    "types": [],
+    "types": ["node"],
+    "typeRoots": ["../node_modules/@types"],
```
Comment (DmitryAstafyev) [inline application/client.core/src/tsconfig.app.json:6]: I would not do this as soon as our client isn't node application

**Review by DmitryAstafyev — CHANGES_REQUESTED** [review]

**Comment by DmitryAstafyev** [comment]
Maybe this issue is related to your local configuration?

**Comment by fur-gan** [comment]
> Maybe this issue is related to your local configuration?

That could be, I will check

**Comment by fur-gan** [comment]
Yeah, seems like this was also an issue of my internal settings. It's related to the path not being recognized with '/' and works as soon as I replace the '/' with '\\'

**Pull request closed without merging**